tasks/repo/yum.yml

```diff
@@ -5,7 +5,7 @@
 
 - name: Add the official sensu repository
   copy:
-    src='../../files/{{ sensu_yum_repo }}.repo'
```

Comment:
Hey, thanks very much for your PR!

Could you please put this change in a separate commit?

Otherwise it looks good to me! 👍